### PR TITLE
Add a method for ceiling to pixel boundaries

### PR DESCRIPTION
**Pull request by mrobinson**
This will be used in Servo to make pixel aligned bounding rectangles.

### CODE REVIEW

**Comment by mrobinson** [comment]
@gw or @SimonSapin r?

File: src/app_unit.rs [hunk 1]
```diff
@@ -123,6 +123,12 @@ impl Au {
         self.0 / AU_PER_PX
     }
 
+    /// Ceil this app unit to the appropriate pixel boundaryjand return it.
```
Comment (mbrubeck) [inline src/app_unit.rs]: typo ("boundaryjand")

**Comment by mbrubeck** [comment]
r=mbrubeck with the typo fixed

**Pull request merged**